package.json

```diff
@@ -2,26 +2,38 @@
   "name": "sgid-test-app",
   "version": "2.0.0",
   "description": "This is an example node application that implements sgID's OAuth2 API.",
-  "main": "index.js",
+  "main": "dist/index.js",
   "engines": {
     "node": ">=16.16"
   },
   "dependencies": {
-    "@opengovsg/sgid-client": "^1.0.4",
+    "@opengovsg/sgid-client": "github:opengovsg/sgid-client#test/v2",
```

Comment:
should use published version?

Reply:
Resolved in [6a21f7a](https://github.com/opengovsg/sgid-test-app/pull/94/commits/6a21f7abfb16f44d9ca5bc9fcf4c6c4e26fd6910).